CMakeLists.txt

```diff
@@ -20,14 +20,20 @@ find_package(Eigen3 3.3    REQUIRED NO_MODULE)
 find_package(Python        REQUIRED COMPONENTS Development)
 find_package(nlohmann_json REQUIRED)
 
+# if cmake install prefix is not set, and conda env is activated, use the
+# conda env as the install directory
+if(DEFINED ENV{CONDA_PREFIX} AND CMAKE_INSTALL_PREFIX STREQUAL "/usr/local")
+ set(CMAKE_INSTALL_PREFIX $ENV{CONDA_PREFIX})
+endif()
+
 # Library setup
 add_library(ale SHARED
             ${CMAKE_CURRENT_SOURCE_DIR}/src/ale.cpp
             ${CMAKE_CURRENT_SOURCE_DIR}/src/Rotation.cpp)
 set(ALE_BUILD_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/include/")
 set(ALE_HEADERS "${ALE_BUILD_INCLUDE_DIR}/ale.h"
                 "${ALE_BUILD_INCLUDE_DIR}/Rotation.h")
-set(ALE_INSTALL_INCLUDE_DIR "include/ale")
+set(ALE_INSTALL_INCLUDE_DIR "include/")
```

Comment:
Why are you changing the directory we are installing into? Some of our headers have very common names that may clash with other packages if they are installed directly into the include directory.

Reply:
:+1: This can be reverted